org-formation/organization.yaml

```diff
@@ -287,6 +288,20 @@ Organization:
         budget-alarm-threshold: 20000
         budget-alarm-threshold-email-recipient: aws-nlpsandbox@sagebase.org
 
+  IncludeAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: IncludeAccount
```

Comment:
This is a very generic name.  Could we come up with something a little more specific that includes some indication whether it's gonna be a dev or prod account?

Reply:
Since INCLUDE is the grant name, do you recommend something like `IncludeProdAccount`?

Reply:
If we can't be more creative then I guess that's fine.  Please also make AccountName the same as Alias.  I guess we like the `org-sagebase-*` convention.